client/ayon_resolve/plugins/publish/collect_editorial_package.py

```diff
@@ -1,88 +1,56 @@
-import json
-
 import pyblish.api
 
 import ayon_api
-from ayon_api import get_task_by_id
 
-from ayon_resolve.api import lib
+from ayon_resolve.api import lib, constants
 
 
-class EditorialPackageInstances(pyblish.api.ContextPlugin):
+class EditorialPackageInstances(pyblish.api.InstancePlugin):
     """Collect all Track items selection."""
 
     order = pyblish.api.CollectorOrder - 0.49
     label = "Collect Editorial Package Instances"
     families = ["editorial_pkg"]
 
-    def process(self, context):
-        project_name = context.data["projectName"]
+    def process(self, instance):
+        project_name = instance.context.data["projectName"]
         self.log.info(f"project: {project_name}")
 
+        media_pool_item_id = instance.data["media_pool_item_id"]
```

Comment:
You're better of here doing:
```suggestion
        media_pool_item = instance.data["transientData"]["timeline_item"]
```

However, the usage of `timeline_item` here is maybe confusing? Because the media pool item isn't a timeline item, is it? So we should maybe also refactor that transient data variable name?

Reply:
I've renamed it to `timeline_pool_item`